switch_model/tools/graphing/main.py

```diff
@@ -0,0 +1,466 @@
+import importlib, traceback
+import os
+import sys
+from typing import List, Dict, Tuple
+
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+import seaborn as sns
+import matplotlib
+
+from switch_model.utilities import StepTimer, get_module_list
+
+original_working_dir = os.getcwd()
+
+
+class GraphDataFolder:
```

Comment:
Are you planning to change this in different modules? I think both of these are constant abroad, maybe we should include them in the utils or in the __init__ of the switch_model.

Reply:
I've added an underscore to indicate it's only meant to be used in this module. When graphing in another module it's accessible through `tools.folders.INPUTS` or `tools.folders.OUTPUTS`. I don't know of another good way to create such constants without making a class.